conf/zapiperf/cdot/9.8.0/rwctx.yaml

```diff
@@ -0,0 +1,24 @@
+name:                     Rwctx
+query:                    rw_ctx
+object:                   rw_ctx
+
+counters:
+  - cifs_giveups
+  - cifs_rewinds
+  - instance_uuid
+  - nfs_giveups
+  - nfs_rewinds
+  - node_name          => node
+  - qos_flowcontrol
+  - qos_giveups
+
+plugins:
+  LabelAgent:
+    split:
+      # instance_uuid  umeng-aff300-06:kernel:rw_ctx
+      - instance_uuid `:` ,type,
```

Comment:
Is type always kernel? If so, do you think it is worth including?

Reply:
Not sure that it will be always `kernel`, doesn't have strong feeling to add it in template, so removing for now.